contentcuration/contentcuration/api.py

```diff
@@ -13,13 +13,31 @@
 from django.db import transaction
 import models
 
+def recurse(node, level=0):
+    print ('\t' * level), node.id, node.lft, node.rght, node.title
+    for child in ContentNode.objects.filter(parent=node).order_by('sort_order'):
+        recurse(child, level + 1)
+
+def clean_db():
```

Comment:
Not sure when the best place to call this would be, but would help with a lot of the extraneous data